testutil/testdatahandler.go

```diff
@@ -0,0 +1,105 @@
+package testutil
+
+import (
+	"path/filepath"
+)
+
+const (
+	TestDataFolderName     = "dcrdata-testdata"
+	TestDescriptorFileName = "test-description.json"
+)
+
+type TestDataHandler struct {
+	expected TestDescriptorJson
```

Comment:
I don't see the need for `TestDataHandler`.  The methods can have `(*TestDescriptorJson)` receivers.